en/user-guide/How-To/index.rst

```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
 
 * If necessary define the horizontal/vertical alignment of your label-coordinates in the placement of the label definition.
 
+Collecting a hydr_geometry (corresponds to a basin geometry in MikeUrban)
+--------------------------
+Note:
+
+* Wastewater structures with a hydr_geometry have to be defined as special structures (and not as standard manholes).
+
+* The detail geometry can be graphically drawn with the **Digitize action**.
+
+Action:
+
+1. Select the wastewater structure with the i-button
+
+2. Select the wastewater nodes tab in the Feature Attributes window
+
+3. Select a hydr_geometry in the fk_hydr_geometry field or use the + button to create a new hydr_geometry.
+
+.. figure:: images/hydr_geometry1.jpg
+
+4. Enter a description in the hydr_geometry - Feature Attributes window. This name is also the name of the table in MikeUrban.
+
+5. Before you can create a hydr: geom_relation, the new record hydro_geometry must be saved: -> click OK
```

Comment:
5. Before you can create a hydr: geom_relation, the new record hydr_geometry must be saved: -> click OK